app/serializers/field_serializer.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class FieldSerializer < ActiveModel::Serializer
+  attributes :name, :result
+
+  def name
+    if object.name.present?
+      object.name.parameterize.underscore
+    else
+      object.name
+    end
+  end
+
+  def result
+    object.result
+  end
```

Comment:
no need to defined this method. It should implicitly delegate `result` to `object`

Reply:
Thank you for explanation.